vendor/bat-native-ledger/src/bat/ledger/internal/legacy/static_values.h

```diff
@@ -3,30 +3,14 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVELEDGER_LEGACY_STATIC_VALUES_H_
-#define BRAVELEDGER_LEGACY_STATIC_VALUES_H_
+#ifndef BRAVE_VENDOR_BAT_NATIVE_LEDGER_SRC_BAT_LEDGER_INTERNAL_LEGACY_STATIC_VALUES_H_
+#define BRAVE_VENDOR_BAT_NATIVE_LEDGER_SRC_BAT_LEDGER_INTERNAL_LEGACY_STATIC_VALUES_H_
 
-#define YOUTUBE_MEDIA_TYPE              "youtube"
-#define TWITCH_MEDIA_TYPE               "twitch"
-#define TWITTER_MEDIA_TYPE              "twitter"
-#define REDDIT_MEDIA_TYPE               "reddit"
-#define VIMEO_MEDIA_TYPE                "vimeo"
-#define GITHUB_MEDIA_TYPE               "github"
-#define YOUTUBE_PROVIDER_URL            "https://www.youtube.com/oembed"
-#define TWITCH_PROVIDER_URL             "https://api.twitch.tv/v5/oembed"
-#define VIMEO_PROVIDER_URL              "https://vimeo.com/api/oembed.json"
-#define YOUTUBE_TLD                     "youtube.com"
-#define TWITCH_TLD                      "twitch.tv"
-#define TWITTER_TLD                     "twitter.com"
-#define REDDIT_TLD                      "reddit.com"
-#define VIMEO_TLD                       "vimeo.com"
-#define GITHUB_TLD                      "github.com"
-#define TWITCH_VOD_URL                  "https://www.twitch.tv/videos/"
-#define REDDIT_USER_URL                 "https://www.reddit.com/user/"
-#define OLD_REDDIT_DOMAIN               "old.reddit.com"
-#define MEDIA_DELIMITER                 "_"
+#define YOUTUBE_MEDIA_TYPE "youtube"
+#define GITHUB_MEDIA_TYPE "github"
+#define YOUTUBE_PROVIDER_URL "https://www.youtube.com/oembed"
+#define YOUTUBE_TLD "youtube.com"
+#define GITHUB_TLD "github.com"
```

Comment:
Could we name these `_DOMAIN` instead of `_TLD` just to be totally clear going forward?

Reply:
Great idea - done!